README.md

```diff
@@ -173,46 +174,41 @@ qiime qemistree get-classyfire-taxonomy \
   --i-feature-data merged-feature-data.qza \
   --o-classified-feature-data classified-merged-feature-data.qza
 ```
-By default, Qemistree will use `ms2_smiles` to make chemical taxonomy assignments. When MS2 matches are not available, `csi_smiles` will be used. the column `annotation_type` in `classified-merged-feature-data.qza` specifies if the taxonomic assignment was done using CSI:FingerID predictions or MS/MS library matches.
+By default, Qemistree will use `ms2_smiles` to make chemical taxonomy assignments. When MS2 matches are not available, `csi_smiles` will be used. The column `structure_source` in `classified-merged-feature-data.qza` specifies if the taxonomic assignment was done using CSI:FingerID predictions or MS/MS library matches.
 Lastly, Qemistree includes some utility functions that are most useful if users would like to visualize the molecular hierarchy generated above.
 
 1. Prune molecular hierarchy to keep only the molecules with annotations.
 
 ```bash
 qiime qemistree prune-hierarchy \
   --i-feature-data classified-merged-feature-data.qza \
-  --p-column subclass \
+  --p-column '#featureID' \
   --i-tree merged-qemistree.qza \
-  --o-pruned-tree merged-qemistree-subclass.qza
+  --o-pruned-tree merged-qemistree-featureID.qza
 ```
 
-Users can choose one of the following data columns (`--p-column`) for pruning: 'kingdom', 'superclass', 'class', 'subclass', 'direct_parent', and 'smiles'. All features with no data in this column will be removed from the phylogeny.
+Users can choose any of the data columns (`--p-column`) that are in the `classified-merged-feature-data.qza` file for pruning. For e.g. '#FeatureID','kingdom', 'superclass', 'class', 'subclass', 'direct_parent', and 'smiles'. All features with no data in this column will be removed from the phylogeny. **Note:** pruning by '#FeatureID' in this case will technically not remove any of the features as they all should have this form of annotation. The use of this columns becomes useful for representing unclassified features on the tree as well.

```

Comment:
I think it is better to give an example where pruning will actually remove features. Let us give an example when users use the column `smiles` for pruning.

Reply:
Hi Priya! Ok cool! I made these changes as well as added information on how to create the grouped feature table